repository/model/project_member.go

```diff
@@ -0,0 +1,29 @@
+package model
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type CreateProjectMemberInput struct {
+	UserID      uuid.UUID `json:"user_id"`
+	ProjectID   uuid.UUID `json:"project_id"`
+	ProjectRole string    `json:"project_role"`
+	CreatedAt   time.Time `json:"created_at"`
+	UpdatedAt   time.Time `json:"updated_at"`
+}
+
+type ProjectMember struct {
+	UserID      uuid.UUID `json:"user_id"`
+	ProjectID   uuid.UUID `json:"project_id"`
+	ProjectRole string    `json:"project_role"`
+	UserData    UserData  `json:"users"`
+	CreatedAt   time.Time `json:"created_at"`
+	UpdatedAt   time.Time `json:"updated_at"`
+}
```

Comment:
And when reading data from database, user data will always be fetched. 

Supabase (postgres) API response:

![Snímek obrazovky 2024-04-27 v 12 05 23](https://github.com/jan-zabloudil/release-manager/assets/81581922/98fd7a31-be9d-4f79-be9b-babc02377f9e)